tests/job_applications/test_reject_job_applications_after_delay.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import io
+import logging
+from unittest.mock import patch
+
+import pytest
+from django.core.management import call_command
+from django.utils import timezone
+
+from itou.job_applications.enums import (
+    AUTO_REJECT_JOB_APPLICATION_DELAY,
+    AUTO_REJECT_JOB_APPLICATION_STATES,
+    JobApplicationState,
+    RefusalReason,
+)
+from itou.job_applications.models import JobApplication
+from tests.job_applications.factories import JobApplicationFactory
+from tests.users.factories import JobSeekerFactory
+
+
+auto_answer = (
+    "Votre candidature a été automatiquement déclinée car elle n’a pas reçu de réponse depuis plus de 2 mois.\n"
+    "\n"
+    "Si vous êtes toujours en recherche d’emploi, nous vous invitons à poursuivre vos démarches. Pour maximiser vos "
+    "chances de retour à l’emploi, n’hésitez pas à vous faire accompagner par un prescripteur habilité (France Travail"
+    ", Mission Locale, Cap emploi, Service social du Conseil départemental…).\n"
+    "\n"
+    "Pour trouver les prescripteurs habilités présents sur votre territoire, consultez notre moteur de recherche : https://emplois.inclusion.beta.gouv.fr/search/prescribers/results\n"
+    "\n"
+    "Nous vous souhaitons une pleine réussite dans vos démarches.\n"
+)
+
+
+@pytest.mark.parametrize("state", AUTO_REJECT_JOB_APPLICATION_STATES)
+def test_reject_job_applications_after_delay(state, django_capture_on_commit_callbacks, mailoutbox):
+    job_seeker = JobSeekerFactory()
+    JobApplicationFactory.create_batch(
+        2,
+        job_seeker=job_seeker,
+        state=state,
+        updated_at=timezone.now() - AUTO_REJECT_JOB_APPLICATION_DELAY - datetime.timedelta(days=1),
```

Comment:
```suggestion
        updated_at=timezone.now() - AUTO_REJECT_JOB_APPLICATION_DELAY,
```

A priori inutile vu que l'on filtre sur un `__lte`

Reply:
C'est important pour l'ordre de tri des `job_applications` dans le test.